symengine/lib/symengine_wrapper.pyx

```diff
@@ -2160,6 +2166,14 @@ cdef class MutableDenseMatrix(DenseMatrixBase):
         for k in range(0, self.cols):
             self[i, k], self[j, k] = self[j, k], self[i, k]
 
+    def applyfunc(self, f):
```

Comment:
What's the difference between this function and the one above?

We'd also need a `_applyfunc` in MutableDenseMatrix that modifies it in place.